src/python/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -292,8 +292,7 @@ def fuzz(self, target_path, options, reproducers_dir, max_time):
         stats.parse_performance_features(
             log_lines,
             options.strategies,
-            options.arguments,
-            include_strategies=False))
```

Comment:
It's been a while since I've looked at this code and I'm not sure why we explicitly avoided adding strategies here. Any idea why this was added initially or if this would have any negative side-effects that I missed?

Reply:
turns out @oliverchang and I had a conversion on this: https://github.com/google/clusterfuzz/pull/835/files#r313988491 TL;DR that was a temporary feature